tox.ini

```diff
@@ -67,6 +67,11 @@ deps =
 commands =
     sphinx-build -a -E -W -d releasenotes/build/doctrees -b html releasenotes/source releasenotes/build/html
 
+[testenv:prospector]
+usedevelop = True
+deps = -r{toxinidir}/requirements-analysis.txt
```

Comment:
Is this helpful? need more info

Reply:
Was just trying and doesn't seem to help. Will remove for the time being.